pkg/cmd/server/api/validation/master.go

```diff
@@ -277,6 +278,42 @@ func ValidateAssetConfig(config *api.AssetConfig) fielderrors.ValidationErrorLis
 		allErrs = append(allErrs, urlErrs...)
 	}
 
+	for _, jsFile := range config.ExtensionJS {
+		if jsFileErrors := ValidateFile(jsFile, "extensionJS"); len(jsFileErrors) > 0 {
```

Comment:
you can just append the results without checking length... `append(list, []...)` is a no-op